custom_components/intellicenter/light.py

```diff
@@ -98,7 +94,6 @@ def __init__(
         self._extra_state_attributes = [USE_ATTR]
 
         self._features = 0
-        self._supported_color_modes = ColorMode.ONOFF
```

Comment:
Removing this line may fix one problem but the original still needs fixing. The correct fix is:

You can see my fix here: https://github.com/dwradcliffe/intellicenter/commit/03bc6a8511f75ce90121d63054bd4aef08b4d32b